FirebaseDynamicLinks/Sources/FIRDynamicLinks.m

```diff
@@ -440,7 +440,11 @@ - (nullable FIRDynamicLink *)dynamicLinkFromCustomSchemeURL:(NSURL *)url {
       }
     }
   }
-  mainQueueCompletion(nil, nil);
+  mainQueueCompletion(
+      nil, [NSError errorWithDomain:@""
+                               code:0
+                           userInfo:[NSDictionary dictionaryWithObject:@"Failed to parse!"
+                                                                forKey:@"Reason"]]);
```

Comment:
This should be [NSLocalizedDescriptionKey](https://developer.apple.com/documentation/foundation/nslocalizeddescriptionkey?language=objc)

Reply:
Changed to "NSLocalizedFailureReasonErrorKey", let me know if that is good?